requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask==2.0.2
 gunicorn
+werkzeug==2.2.2
```

Comment:
does the sample code use this package directly.. or is it a dependency of other package? If so, should we try to upgrade Flask to use latest version etc instead?

Reply:
@vivekjilla the given package is a dependency of flask, so I'll try to upgrade the flask with the newer version and check what happens.